sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/AssetUploadUriResult.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> The result of obtaining the upload location for an asset. </summary>
+    [CodeGenModel("UploadLocation")]
+    public partial class AssetUploadUriResult
+    {
+        /// <summary> Initializes a new instance of GetAssetUploadUriResult. </summary>
+        /// <param name="inputAssetUriString"> The blob upload URI in storage where a model should be uploaded to the service for conversion. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="inputAssetUriString"/> is null. </exception>
+        internal AssetUploadUriResult(string inputAssetUriString)
+        {
+            if (inputAssetUriString == null)
+            {
+                throw new ArgumentNullException(nameof(inputAssetUriString));
+            }
+
+            UploadUri = new Uri(inputAssetUriString);
+        }
+
+        /// <summary> The blob upload URI in storage where a model should be uploaded to the service for conversion. </summary>
+        public Uri UploadUri { get; }
+
+        [CodeGenMember("InputAssetUri")]
+        internal string UploadUriString { get => UploadUri.AbsoluteUri; }
```

Comment:
Missing docs?

Reply:
Internal properties, corresponding to auto-generated code, wrapped by or wrapping more meaningful data which is documented, should I still document these internal properties?

Reply:
It doesn't hurt. I thought I saw other `internal` members being documented.